app/components/login/login.component.ts

```diff
@@ -1,14 +1,51 @@
-import { Component } from '@angular/core';
-
-import { LoginService } from '../../services/login/login.service';
+import {NgForm} from '@angular/forms';
+import {LoginService} from '../../services/login/login.service';
+import {Component, OnInit} from '@angular/core';
+import {FormGroup, FormBuilder, Validators} from '@angular/forms';
+import { Router, ActivatedRoute } from '@angular/router';
+import Swal from 'sweetalert2'
+import {RegistrationComponent} from "../registration/registration.component";
 
 @Component({
-  selector: 'login',
-  templateUrl: './login.html',
-  providers: [LoginService],
-  styleUrls: ['./login.css']
+    selector: 'login',
+    templateUrl: './login.html',
+    providers: [LoginService],
+    styleUrls: ['./login.css']
 })
 
-export class LoginComponent {
-  constructor() { }
+export class LoginComponent implements OnInit{
+    user: any = {};
```

Comment:
Left to right in Hebrew is annoying so English it is.
I think that instead of `any` its better to define an [interface](https://angular-2-training-book.rangle.io/handout/features/interfaces.html) that declares exactly its type:
```
interface User {
    email: { ... },
    password: { ... }
}
```
With all the: errors, required, touched...

And then you declare user's type to be User:
`user: User = { ... };`

And by the way I prefer boolean parameters names to start with: is, has, will etc.
So instead of `touched` for example `isChanged` but your call.